lsp/server/src/validateDocument.ts

```diff
@@ -26,26 +26,25 @@ export async function validateDocument(
     document: TextDocument,
     extensionName: string
 ): Promise<Diagnostic[]> {
+    const { uri } = document;
 
     let results: Diagnostic[] = [];
 
     if (document.languageId === 'javascript') {
         // handles JS rules
-        const jsDiagnostics = await validateJs(
-            setting,
-            document
-        );
-        
+        const jsDiagnostics = await validateJs(setting, document);
+
         // handle graphql rules
-        const graphqlDiagnostics = await validateGraphql(
-            setting,
-            document
-        );
+        const graphqlDiagnostics = await validateGraphql(setting, document);
 
         results = results.concat(jsDiagnostics, graphqlDiagnostics);
     }
 
     if (document.languageId === 'html') {
+        const diagnostics = await validateHtml(setting, document);
+        results = results.concat(diagnostics);
+        var aaa = 0;
```

Comment:
nitpick: could be removed

Reply:
Argh. You are right, it should be removed. Left over from debugging the code.